modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/EpisodeManagerImpl.kt

```diff
@@ -1098,4 +1099,18 @@ class EpisodeManagerImpl @Inject constructor(
 
             return@withContext newDownloadUrl ?: episode.downloadUrl
         }
+
+    override suspend fun selectChapterIndexForEpisode(chapterIndex: Int, episode: PodcastEpisode) {
+        val deselectedChapterIndices = episode.deselectedChapters ?: emptyList()
+        if (!deselectedChapterIndices.contains(chapterIndex)) return
+        episode.deselectedChapters = ChapterIndices(deselectedChapterIndices.toMutableList().apply { remove(chapterIndex) })
```

Comment:
Similarly to what you did in `deselectChapterIndexForEpisode` you can use minus operator here.

```suggestion
        episode.deselectedChapters = ChapterIndices(deselectedChapterIndices - chapterIndex)
```